core/unit_test/cuda/TestCuda_Spaces.cpp

```diff
@@ -254,6 +254,12 @@ TEST(cuda, space_access) {
           Kokkos::Impl::HostMirror<Kokkos::CudaHostPinnedSpace>::Space,
           Kokkos::HostSpace>::accessible,
       "");
+#ifdef KOKKOS_ENABLE_CUDA_UVM
+  using uvm_view = Kokkos::View<double *, CudaUVMSpace>;
+  static_assert(std::is_same<uvm_view::HostMirror::execution_space,
+                             DefaultHostExecutionSpace>::value,
```

Comment:
You forgot to specify the `Kokkos::` namespace for `CudaUVMSpace` and `DefaultHostExecutionSpace`.  This does not compile.

Reply:
Yeah, saw that. All of our GPU testbeds are down right now so I have to test by pushing new commits...